common.tf

```diff
@@ -129,6 +130,13 @@ module "ingest_raw_cache_bucket" {
   kms_key_arn = module.dr2_kms_key.kms_key_arn
 }
 
+module "sample_files_bucket" {
```

Comment:
Event though this is for sample files, should we add a bucket policy?

Reply:
It's got one. I updated the [da-terraform-modules](https://github.com/nationalarchives/da-terraform-modules/blob/main/s3/templates/default_bucket_policy.json.tpl) to add a default bucket policy if there isn't one set. It just makes sure that all requests are over SSL.